CefSharp.Example/CefSharp.Example.netcore.csproj

```diff
@@ -17,6 +17,15 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
There's no need for the conditional property group, simply add the setting once to the block above.